yarn.lock

```diff
@@ -2090,14 +2100,28 @@
     isomorphic-fetch "^2.2.1"
     websocket "^1.0.29"
 
+"@polkadot/rpc-provider@^0.91.0-beta.6":
+  version "0.91.0-beta.6"
+  resolved "https://registry.yarnpkg.com/@polkadot/rpc-provider/-/rpc-provider-0.91.0-beta.6.tgz#43accc2c6d097baa9f47e35de3b929e81502ce47"
+  integrity sha512-gn6ASGpG0G0aHh1XXhbr1snWt/uuzurCdMxzoDo1J3mmXcQ/1DeFnPyiyLmayPiX9nTYAWWZay3LTikMCGpaXw==
+  dependencies:
+    "@babel/runtime" "^7.5.5"
+    "@polkadot/api-metadata" "^0.91.0-beta.6"
+    "@polkadot/util" "^1.1.1"
+    "@polkadot/util-crypto" "^1.1.1"
+    "@types/nock" "^10.0.3"
+    eventemitter3 "^4.0.0"
+    isomorphic-fetch "^2.2.1"
+    websocket "^1.0.29"
+
 "@polkadot/ts@^0.1.68":
   version "0.1.68"
   resolved "https://registry.yarnpkg.com/@polkadot/ts/-/ts-0.1.68.tgz#ef44eeac95d736ad60dbf83c76acfd6d192c5454"
   integrity sha512-vcnX9cDQJ959QNFlHj7QnKeShBr9GIi4vpVrISaIZpmBuF2d3vO0NMu3Fzn/aNvRFJdmfJoJFX55UdfAvAYC5Q==
   dependencies:
     "@types/chrome" "^0.0.86"
 
-"@polkadot/types@^0.91.0-beta.2":
+"@polkadot/types@^0.91.0-beta.2", "@polkadot/types@^0.91.0-beta.6":
```

Comment:
Would like to avoid this type of thing as much as possible - since `@polkadot/api` already depends on `rpc-provider` we really don't need to pull it in explicitly. (As soon as things start mismatching, we may have issues)

So would suggest reverting these

https://github.com/polkadot-js/extension/pull/88/files#diff-ada64edcbc9054dffd93143b631be6ebR10
https://github.com/polkadot-js/extension/pull/88/files#diff-2536f4ac6a738a37164af87d5b55499bR12

Reply:
Well, didn't know you could import a dependency's dependencies without having them be in your `package.json`. Learned something new today

:+1: